### ISSUE TITLE
java.lang.NullPointerException Attempt to invoke virtual method 'int android.graphics.Bitmap.getWidth()' on a null object reference

### ISSUE THREAD

**Issue by dgcoffman** [Buglife/buglife-android#78]
On Android 11, submitting a report w/ a screenshot attached crashes with

```
java.lang.NullPointerExceptionMainActivity
Attempt to invoke virtual method 'int android.graphics.Bitmap.getWidth()' on a null object reference

Thread 2 - main
    AttachmentAdapter.java:90 com.buglife.sdk.AttachmentAdapter.scaleBitmapForThumbnail
    AttachmentAdapter.java:81 com.buglife.sdk.AttachmentAdapter.getView
    AbsListView.java:2387 android.widget.AbsListView.obtainView
    ListView.java:1417 android.widget.ListView.measureHeightOfChildren
    ListView.java:1323 android.widget.ListView.onMeasure
    View.java:25466 android.view.View.measure
    ViewGroup.java:6957 android.view.ViewGroup.measureChildWithMargins
    LinearLayout.java:1552 android.widget.LinearLayout.measureChildBeforeLayout
    LinearLayout.java:842 android.widget.LinearLayout.measureVertical
    LinearLayout.java:721 android.widget.LinearLayout.onMeasure
    View.java:25466 android.view.View.measure
    ViewGroup.java:6957 android.view.ViewGroup.measureChildWithMargins
    FrameLayout.java:194 android.widget.FrameLayout.onMeasure
    ContentFrameLayout.java:143 androidx.appcompat.widget.ContentFrameLayout.onMeasure
    View.java:25466 android.view.View.measure
    ViewGroup.java:6957 android.view.ViewGroup.measureChildWithMargins
    LinearLayout.java:1552 android.widget.LinearLayout.measureChildBeforeLayout
    LinearLayout.java:842 android.widget.LinearLayout.measureVertical
    LinearLayout.java:721 android.widget.LinearLayout.onMeasure
    View.java:25466 android.view.View.measure
    ViewGroup.java:6957 android.view.ViewGroup.measureChildWithMargins
    FrameLayout.java:194 android.widget.FrameLayout.onMeasure
    View.java:25466 android.view.View.measure
    ViewGroup.java:6957 android.view.ViewGroup.measureChildWithMargins
    LinearLayout.java:1552 android.widget.LinearLayout.measureChildBeforeLayout
    LinearLayout.java:842 android.widget.LinearLayout.measureVertical
    LinearLayout.java:721 android.widget.LinearLayout.onMeasure
    View.java:25466 android.view.View.measure
    ViewGroup.java:6957 android.view.ViewGroup.measureChildWithMargins
    FrameLayout.java:194 android.widget.FrameLayout.onMeasure
    DecorView.java:747 com.android.internal.policy.DecorView.onMeasure
    View.java:25466 android.view.View.measure
    ViewRootImpl.java:3397 android.view.ViewRootImpl.performMeasure
    ViewRootImpl.java:2228 android.view.ViewRootImpl.measureHierarchy
    ViewRootImpl.java:2486 android.view.ViewRootImpl.performTraversals
    ViewRootImpl.java:1952 android.view.ViewRootImpl.doTraversal
    ViewRootImpl.java:8171 android.view.ViewRootImpl$TraversalRunnable.run
    Choreographer.java:972 android.view.Choreographer$CallbackRecord.run
    Choreographer.java:796 android.view.Choreographer.doCallbacks
    Choreographer.java:731 android.view.Choreographer.doFrame
    Choreographer.java:957 android.view.Choreographer$FrameDisplayEventReceiver.run
    Handler.java:938 android.os.Handler.handleCallback
    Handler.java:99 android.os.Handler.dispatchMessage
    Looper.java:223 android.os.Looper.loop
    ActivityThread.java:7656 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    RuntimeInit.java:592 com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run
    ZygoteInit.java:947 com.android.internal.os.ZygoteInit.main
```

We have
```
    implementation 'com.buglife.sdk:buglife-android:1.5.1'
```

**Comment by dgcoffman** [comment]
@schukin @danhd123 Is this maintained?

**Comment by schukin** [comment]
@dgcoffman thanks for reporting this! We'll be shipping an update next week that should resolve this issue.

**Comment by danhd123** [comment]
Fixed in 722008ee. New binary pending.
